### PR TITLE
Update readme to include more information about image availability

### PR DESCRIPTION
**Pull request by maxim-lobanov**
# Description
Add two sections to README:
- ***What images are available for GitHub Actions and Azure DevOps?***
- ***What image version is used in my build?***

Corrected **Regular Weekly Rhythm**

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by al-cheb — APPROVED** [review]

**Review by AlenaSviridenko — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -18,14 +18,17 @@ For general questions about using the virtual environments or writing your Actio
...
 The MacOS 11.0 virtual environment is currently provided as a preview only.
 The "macos-latest" YAML workflow label still uses the MacOS 10.15 virtual environment.
 ```
 
+***What images are available for GitHub Actions and Azure DevOps?***
+The availability of images for GitHub Actions and Azure DevOps is different. See documentation for more details:  
+- [GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources)  
+- [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software)  
+
+***What image version is used in my build?*** Usually, image deployment takes 3-4 days and documentation in `main` branch is updated only when deployment is finished. To find out which image version and what software versions are used in specific build, see logs of `Set up job` (GitHub Actions) or `Initialize job` (Azure DevOps) steps.
```
Comment (miketimofeev) [inline README.md:30]: in **the** `main`
in **a** specific build

Comment (miketimofeev) [inline README.md:30]: is updated only -> is only updated

Comment (miketimofeev) [inline README.md:30]: comma before `and documentation`

Comment (miketimofeev) [inline README.md:30]: see logs of `Set up job` (GitHub Actions) or `Initialize job` (Azure DevOps) steps -> see `Set up job` (GitHub Actions) or `Initialize job` (Azure DevOps) step log

**Pull request merged**